examples/jsm/nodes/core/NodeUtils.js

```diff
@@ -1,72 +1,76 @@
 import { Color, Matrix3, Matrix4, Vector2, Vector3, Vector4 } from 'three';
 
-export const getCacheKey = ( object ) => {
+export function getCacheKey( object )  {
 
 	let cacheKey = '{';
 
 	if ( object.isNode === true ) {
 
-		cacheKey += `uuid:"${ object.uuid }",`;
+		cacheKey += `uuid:"${ object.uuid }"`;
 
 	}
 
-	for ( const property of getNodesKeys( object ) ) {
-
-		const node = object[ property ];
+	for ( const { prop, childNode } of getNodeChildren( object ) ) {
 
 		// @TODO: Think about implement NodeArray and NodeObject.
 
-		if ( Array.isArray( node ) ) {
+		let childCacheKey = getCacheKey( childNode );
+		if ( ! childCacheKey.includes( ',' ) ) childCacheKey = childCacheKey.slice( childCacheKey.indexOf( '"' ), childCacheKey.indexOf( '}' ) );
```

Comment:
This line just converts `childCacheKey` to just uuid if it has only it (i.e. something like `{uuid:"..."}` to `"..."`).